Brokerages/Bitfinex/BitfinexSubscriptionManager.cs

```diff
@@ -462,9 +485,6 @@ private void OnSnapshot(BitfinexWebSocketWrapper webSocket, string channelId, st
                     case "book":
                         ProcessOrderBookSnapshot(channel, entries);
                         return;
-                    case "trades":
-                        ProcessTradesSnapshot(channel, entries);
```

Comment:
Wondering why was this a bug?

Reply:
When successfully subscribing to a symbol, the exchange sends a snapshot containing the latest 20-40 ticks (can't remember the exact number), this is historical data which should not be emitted as live ticks.